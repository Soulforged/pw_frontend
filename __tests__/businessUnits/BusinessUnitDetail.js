//@flow
import React from "react";
import BusinessUserDetail from "src/businessUnits/BusinessUnitDetail";
import renderer from "react-test-renderer";

it("details matches snapshot", () => {
  const rendered = renderer.create(<BusinessUserDetail getBusinessUnit={jest.fn()} />).toJSON();
  expect(rendered).toMatchSnapshot();
});

it("details matches snapshot with data", () => {
  const rendered = renderer.create(<BusinessUserDetail getBusinessUnit={jest.fn()} item={{ id: 1 }} />).toJSON();
  expect(rendered).toMatchSnapshot();
});