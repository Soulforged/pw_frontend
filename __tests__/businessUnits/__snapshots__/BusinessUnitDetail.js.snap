// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`details matches snapshot 1`] = `
<div
  className="details pnls"
  id="main-pnl"
>
  <div
    className="modal-p details"
  >
    Not found
  </div>
</div>
`;

exports[`details matches snapshot with data 1`] = `
<div
  className="details pnls"
  id="main-pnl"
>
  <div
    className="details"
  >
    <h5>
      Details for 
      1
    </h5>
    <button
      className="btn btn-default pull-right"
      onClick={[Function]}
      title="Edit"
    >
      <span
        className="edit fa fa-pencil theme"
      />
      Edit user
    </button>
    <div
      className="row"
    >
      <div
        className="col-sm-3"
      >
        Name:
      </div>
      <div
        className="col-sm-3"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="col-sm-3"
      >
        Status:
      </div>
      <div
        className="col-sm-3"
      >
        <span
          className="td-error"
        >
          <i
            className="fa fa-circle"
          />
          Inactive
        </span>
      </div>
    </div>
  </div>
</div>
`;
